react/features/chat/components/web/MessageRecipient.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import React from 'react';
+
+import { translate } from '../../../base/i18n';
+import { Icon, IconCancelSelection } from '../../../base/icons';
+import { connect } from '../../../base/redux';
+
+import AbstractMessageRecipient, {
+    _mapDispatchToProps,
+    _mapStateToProps
+} from '../AbstractMessageRecipient';
+
+/**
+ * Class to implement the displaying of the recipient of the next message.
+ */
+class MessageRecipient extends AbstractMessageRecipient {
+    /**
+     * Implements {@code PureComponent#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const { _privateMessageRecipient } = this.props;
+
+        if (!_privateMessageRecipient) {
+            return null;
+        }
+
+        const { t } = this.props;
+
+        return (
+            <div id = 'chat-recipient'>
+                <span>
+                    { t('chat.messageTo', {
+                        name: _privateMessageRecipient
```

Comment:
I think in another string you called this rcpt, so maybe it's good to be consistent with the variable name.